h2o-py/tests/pydemo_utils/utilsPY.py

```diff
@@ -41,8 +41,11 @@ def ipy_valid_lines(block):
     for line in lines:
         if "import matplotlib.pyplot as plt" in line or "%matplotlib inline" in line:
             import matplotlib
-            matplotlib.use('Agg', warn=False)
-
+            try:
+                matplotlib.use('Agg', warn=False)
```

Comment:
`warn` is no longer supported but it's ok to use it on jenkins since we use older version of matplotlib there.
I added the try except block here to be able to test it locally (I have newer matplotlib) and I didn't want to break anything so I kept `warn=False` just in case we check somewhere for warnings.

https://matplotlib.org/stable/api/matplotlib_configuration_api.html#matplotlib.use